dali/test/python/test_operator_arithmetic_ops.py

```diff
@@ -436,24 +448,26 @@ def test_unary_arithmetic_ops():
 def check_math_function_op(kind, type, op, np_op, shape, get_range, op_desc, eps):
     is_integer = type not in [np.float16, np.float32, np.float64]
     limted_range = get_range(type)
-    iterator = iter(ExternalInputIterator(batch_size, shape, type, kind, limited_range=limted_range))
+    iterator = iter(ExternalInputIterator(batch_size, shape, type, kind,
+                                          limited_range=limted_range))
     pipe = ExprOpPipeline(kind, type, iterator, op, batch_size=batch_size, num_threads=2,
-            device_id=0)
+                          device_id=0)
```

Comment:
```suggestion
    pipe = ExprOpPipeline(kind, type, iterator, op,
                          batch_size=batch_size, num_threads=2, device_id=0)
```

Reply:
I'd like this file compatible with full auto-format and this doesn't work.